web/src/utils/flows.ts

```diff
@@ -0,0 +1,12 @@
+import * as _ from 'lodash';
+import { Record } from '../api/ipfix';
+
+export const mergeFlowReporters = (flows: Record[]): Record[] => {
+  // The purpose of this function is to determine if, for a given [srcip, dstip] couple, we'll look at INGRESS or EGRESS reporter
+  // The assumption is that INGRESS alone, or EGRESS alone, or both of them, always provide a complete visiblity, so we can just pick one of the two.
+  // The logic if then to index flows by src+dest ips, then for each indexed set, keep only the first-found reporter
+  const keyFunc = (r: Record) => r.fields.SrcAddr + '+' + r.fields.DstAddr;
+  const grouped = _.groupBy(flows, keyFunc);
+  const filtersIndex = _.mapValues(grouped, (recs: Record[]) => recs[0].labels.FlowDirection);
+  return flows.filter((r: Record) => r.labels.FlowDirection === filtersIndex[keyFunc(r)]);
+};
```

Comment:
I have doubts about this functions. INGESS or EGRESS alone doesn't report the same amount of packets / bytes

You see the difference by taking the last found reporter for example `recs[recs.length -1]`
![image](https://github.com/netobserv/network-observability-console-plugin/assets/91894519/6a17df6d-f3db-4a7a-b593-c949933c55e0)
![image](https://github.com/netobserv/network-observability-console-plugin/assets/91894519/a44f20fd-01c1-43cf-bf94-8d2fa96b5c58)

Reply:
Can you open a ticket and I will investigate post-merge? There might be some discrepancies related to having sampling, but I need to confirm

Reply:
https://issues.redhat.com/browse/NETOBSERV-1235